ext/CMakeLists.txt

```diff
@@ -117,6 +117,9 @@ add_subdirectory(drutil)
 add_subdirectory(droption)
 add_subdirectory(drcovlib)
 add_subdirectory(drstatecmp)
+if (LINUX AND X86 AND X64)
```

Comment:
Use HAVE_LIBIPT: that's why we have it, to avoid duplicating the complex list of conditions.

Reply:
Done.